packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMainInput.tsx

```diff
@@ -169,6 +169,7 @@ const TransferPanelInputField = React.memo(
         type="text"
         inputMode="decimal"
         placeholder="0"
+        aria-label="Amount input"
```

Comment:
we won't be able to differentiate between the amount1 and amount2 inputs based on this aria-label